lib/sqlalchemy/engine/result.py

```diff
@@ -1223,7 +1243,7 @@ def t(self) -> TupleResult[_TP]:
         """
         return self  # type: ignore
 
-    def tuples(self) -> TupleResult[_TP]:
+    def tuples(self) -> TupleResult[Tuple[Unpack[_Ts]]]:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

can be deprecated, same as `.t`

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
Thanks, done

Reply:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986